database_cleaner.gemspec

```diff
@@ -11,6 +11,8 @@ Gem::Specification.new do |spec|
   spec.homepage    = "https://github.com/DatabaseCleaner/database_cleaner"
   spec.license     = "MIT"
 
+  s.metadata["changelog_uri"] = "https://github.com/DatabaseCleaner/database_cleaner/blob/master/History.rdoc"
```

Comment:
Great idea, but the `s` variable needs to be `spec`. Fix that and I think this is good to merge!